tests/samplers_tests/cmaes_tests/test_sampler.py

```diff
@@ -0,0 +1,108 @@
+import numpy as np
+import optuna
+import pytest
+
+from mock import MagicMock
+from mock import patch
+from optuna.samplers.cmaes.sampler import _initialize_sigma0
+from optuna.samplers.cmaes.sampler import _initialize_x0
+from optuna.testing.distribution import UnsupportedDistribution
+from optuna.testing.sampler import DeterministicRelativeSampler
+
+
+def test_init_cmaes_opts():
```

Comment:
I think we can use the Python3-style type hinting. ditto to other test functions.
```suggestion
def test_init_cmaes_opts() -> None:
```